### ISSUE TITLE
Support for Struts 1.3 framework? 

### ISSUE THREAD

**Issue by rawasthi11** [HotswapProjects/HotswapAgent#205]
Hi Techies,

I think HotSwap agent currently  not supporting Struts framework, But i want to use HSA in my project which is built upon Struts framework. Any idea how to use or what type of change to be made so it start working for struts too. 
Also is there any other software(Open Source) available which can solve my problem ??

**Comment by skybber** [comment]
You are right, there is curently no Struts support in HA and we has no plan to do it. If you realize to create a new plugin for Struts, you will be welcome.
